library/haxe

```diff
@@ -2,50 +2,82 @@ Maintainers: Andy Li <andy@onthewings.net> (@andyli)
 GitRepo: https://github.com/HaxeFoundation/docker-library-haxe.git
 
 Tags: 3.4.2-stretch, 3.4-stretch, 3.4.2, 3.4, latest
+Architectures: amd64
 GitCommit: 718e80d01e38b9e40b4f221c2ed54d7d134d80f7
 Directory: 3.4/stretch
 
 Tags: 3.4.2-jessie, 3.4-jessie
+Architectures: amd64
 GitCommit: 718e80d01e38b9e40b4f221c2ed54d7d134d80f7
 Directory: 3.4/jessie
 
 Tags: 3.4.2-onbuild, 3.4-onbuild
+Architectures: amd64
 GitCommit: 8e6c65a2d72c239e8ddaf5af9157bb146d71016d
 Directory: 3.4/onbuild
 
+Tags: 3.4.2-windowsservercore, 3.4-windowsservercore
+Architectures: windows-amd64
+GitCommit: ba6ec51ec4db8248066b9a294b22bcbfd83a20f4
+Directory: 3.4/windowsservercore
```

Comment:
Can we also have a `Constraints: windowsservercore` like golang.

https://github.com/docker-library/official-images/blob/ed3fb22b631098226528ee7a73350e53cb6dc6fd/library/golang#L18-L22

Reply:
Didn't notice that is required. Just added it back :)